csi/moac/volume.js

```diff
@@ -23,8 +23,10 @@ class Volume {
   // @params {string[]} spec.requiredNodes   Replicas must be on these nodes.
   // @params {number}   spec.requiredBytes   The volume must have at least this size.
   // @params {number}   spec.limitBytes      The volume should not be bigger than this.
+  // @params {string}   spec.protocol        The share protocol for the nexus.
+  // @params  {object}  status               Current properties of the volume
```

Comment:
Optional arg should have `[...]` around it.

Reply:
thanks will do